src/plugins/FingerPrint.js

```diff
@@ -215,30 +223,30 @@ module.exports = class FingerPrint extends OverlayPlugin {
         // buttons fingerprint
         const fingerprintButtons = [
             {
-                cmd: 'recognized',
+                cmd: 'Recognized',
                 message: FINGERPRINT_MESSAGES.toSend.SCAN_RECOGNIZED,
                 onclick: (btn) => {
                     btn.classList.add('gm-loader');
                 },
             },
             {
-                cmd: 'unrecognized',
+                cmd: 'Unrecognized',
                 message: FINGERPRINT_MESSAGES.toSend.SCAN_UNRECOGNIZED,
             },
             {
-                cmd: 'dirty',
+                cmd: 'Dirty sensor',
                 message: FINGERPRINT_MESSAGES.toSend.SCAN_DIRTY,
             },
             {
-                cmd: 'too-fast',
+                cmd: 'Too-fast',
```

Comment:
```suggestion
                cmd: 'Too fast',
```
I guess this should work since there's a replace later